samples/apim-policy/README.md

```diff
@@ -0,0 +1,108 @@
+<!-- markdownlint-disable MD024 -->
+# Azure API Management Policy
+
+This folder contains a sample integration of the [Carbon Aware SDK](https://github.com/Green-Software-Foundation/carbon-aware-sdk)
+with [Azure API Management](https://azure.microsoft.com/en-us/products/api-management/) (APIM)
+[policies](https://learn.microsoft.com/en-us/azure/api-management/api-management-howto-policies).
+
+By integrating the Carbon Aware SDK at the API gateway level
+backend API's and/or downstream consumers are able to do their work
+at optimal carbon intensity times without any knowledge of the
+underlying system that determines those times. This can be especially
+valuable in situations where the integration is with legacy systems
+or systems that cannot be readily modified to include carbon awareness.
+
+This sample includes two use cases:
+
+1. [Request routing](#request-routing)
+2. [Scheduled message delivery](#scheduled-message-delivery)
+
+In both use cases the Carbon Aware Web.Api is accessed behind an
+APIM API to take advantage of response caching. This allows a high volume
+API to use the Carbon Aware Web.Api without concerns of being
+throttled by the underlying services that the Carbon Aware SDK
+uses to provide the forecast data.
+
+## Request routing
+
+The request routing policy sample source can be found here - [request-routing-policy.xml](request-routing-policy.xml).
+
+### Overview
+
+The `request routing` use case involves an APIM policy that routes requests
+based on the current carbon intensity in the target regions/data centers.
+In the sample policy, requests are routed to the [backend API](https://learn.microsoft.com/en-us/azure/api-management/backends)
+in the region with the lowest carbon intensity.
+
+This use case can be implemented in cases where requests are time
+sensitive and cannot take advantage of time shifting for optimal future
+carbon intensity usage.
+
+![Request Routing](images/request-routing.jpg "Request Routing")
```

Comment:
So glad you included the links to these images in the readme, really helps visualize the flow of information.